graaljs/graaljs-gradle-webpack-guide/gradle/wrapper/gradle-wrapper.properties

```
@@ -0,0 +1,6 @@
+#Mon Mar 10 09:26:42 WET 2025
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.10-bin.zip
```

Let's be consistent with what we have in the starter: https://github.com/graalvm/graal-languages-demos/blob/0951909cb64a04262eaa6a27bc5800846e785019/graaljs/graaljs-starter/gradle/wrapper/gradle-wrapper.properties#L3
```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-8.10.1-bin.zip
```